query/metaquery/utils_test.go

```diff
@@ -0,0 +1,29 @@
+package metaquery
+
+import (
+	"reflect"
+	"testing"
+)
+
+type CmdAndArgsExpected struct {
+	cmd  string
+	args []string
+}
+
+func TestGetCmdAndArgs(t *testing.T) {
+	cases := map[string]CmdAndArgsExpected{
+		`.cmd arg1`:             {cmd: ".cmd", args: []string{"arg1"}},
+		`.cmd arg1 arg2`:        {cmd: ".cmd", args: []string{"arg1", "arg2"}},
+		`.cmd "arg1 arg2" arg3`: {cmd: ".cmd", args: []string{"arg1 arg2", "arg3"}},
```

Comment:
haveing "arg1 arg2" is confusing as they are not arg2 and 3 but both arg2 

maybe 
```
.cmd "arg1a arg1b" arg2`
```
or "foo bar" or whatever